tests/cypress/e2e/unit_tests/p1_2_fleet.spec.ts

```diff
@@ -1246,3 +1246,58 @@ describe('Test non-yaml file into bundle.', { tags: '@p1_2'}, () => {
     })
   )
 });
+
+describe('Test Fleet doNotDeploy.', { tags: '@p1_2'}, () => {
+  const key = "key_resources"
+  const value = "deploy_true"
+
+  beforeEach('Cleanup leftover GitRepo or cluster labels if any.', () => {
+    cy.login();
+    cy.visit('/');
+    cy.deleteAllFleetRepos();
```

Comment:
Why this block is needed if we have it at the begining of the file?

Reply:
I will remove it, thanks for catching it.